src/server/client.ts

```diff
@@ -692,6 +692,50 @@ namespace ts.server {
             return response.body.map(entry => this.convertCodeActions(entry, fileName));
         }
 
+        getRefactorDiagnostics(fileName: string, range?: TextRange): RefactorDiagnostic[] {
+            const startLineOffset = this.positionToOneBasedLineOffset(fileName, range.pos);

```

Comment:
Here the `range` is not supposed to be nullable, as we only use it to test the session API `GetRefactorsForRange`. Will update